rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :rentals, :book
```

Comment:
- [ ]  You did an amazing job to create a Rental class, and its instance variables. One thing you can improve is to create both Getters and Setters for` @book`,  `@person` as you did it for` :date`.  You can use attr_accessor to create both setter and getter the same time.